infra/infra-integrations/infra-integrations-email/src/main/java/com/sflpro/notifier/externalclients/email/smtp/SmtpTransportService.java

```diff
@@ -17,6 +21,11 @@
      * @param to
      * @param subject
      * @param body
+     * @param fileAttachments
      */
-    void sendMessageOverSmtp(final String from, final String to, final String subject, final String body);
+    void sendMessageOverSmtp(final String from,
+                             final String to,
+                             final String subject,
+                             final String body,
+                             final List<SpiEmailNotificationFileAttachment> fileAttachments);
```

Comment:
Why the model is in spi package?

Reply:
I needed to use it in `ImmutableTemplatedEmailMessage ` and `ImmutableSimpleEmailMessage` and I thought it would be appropriate to put it in the same package. Should I change the location of the class?

Reply:
ok